tests/test_controller.py

```diff
@@ -290,6 +290,10 @@ def test_libraries(self):
         ct = self.connecticut_state_library
         ks = self.kansas_state_library
         nypl = self.nypl
+        everywhere = self._place(type=Place.EVERYWHERE)
+        ia = self._library(
+            "InternetArchive", "IA", [everywhere], has_email=True
+        )
```

Comment:
To test the `or "Everywhere"` bit I needed to add a library to the test case whose service area was 'everywhere'.